src/docbuilder/rustwide_builder.rs

```diff
@@ -160,8 +160,8 @@ impl RustwideBuilder {
         // NOTE: this ignores the error so that you can still run a build without rustfmt.
         // This should only happen if you run a build for the first time when rustfmt isn't available.
         if let Err(err) = self.toolchain.add_component(&self.workspace, "rustfmt") {
-            log::warn!("failed to install rustfmt: {}", err);
-            log::info!("continuing anyway, since this must be the first build");
+            tracing::warn!("failed to install rustfmt: {}", err);
+            tracing::info!("continuing anyway, since this must be the first build");
```

Comment:
can we import these macros to avoid churn in the future?

Reply:
will do so